core/domain/feedback_services.py

```diff
@@ -245,3 +246,11 @@ def get_all_threads(exploration_id, has_suggestion):
         if thread.has_suggestion == has_suggestion:
             all_threads.append(thread)
     return all_threads
+
+
+def enqueue_feedback_message_email_task(user_id):
+    """Adds task for sending email of feedback messages into taskqueue"""
```

Comment:
Adds a 'send feedback email' task into the task queue.

(Sorry, this is just a nit. For some reason I keep reading this as "sending email ... into taskqueue", which sounds entirely plausible for a few seconds :P)